numpy/linalg/umath_linalg.c.src

```diff
@@ -2103,13 +2111,18 @@ process_@lapack_func@_results(GEEV_PARAMS_t *params)
 
 /**end repeat**/
 
+/*
+ * TODO:
+ * reinstate the CFLOAT, COMPLEX_t, fortran_complex, float, cggev variant
+ * for single floats and use it in eig, eigval
+ */
 
 /**begin repeat
-   #TYPE = CFLOAT, CDOUBLE#
-   #typ = COMPLEX_t, DOUBLECOMPLEX_t#
-   #ftyp = fortran_complex, fortran_doublecomplex#
-   #realtyp = float, double#
-   #lapack_func = cgeev, zgeev#
+   #TYPE = CDOUBLE#
+   #typ = DOUBLECOMPLEX_t#
+   #ftyp = fortran_doublecomplex#
+   #realtyp = double#
+   #lapack_func = zgeev#
```

Comment:
Instead of not generating this code at all, can we generate it inside `#if 0`? Writing these templates can be hard to get write, so it seems a shame to back out the partial work done for floats.